### DISCUSSION TITLE
奇忽局的内容架构模型

### DISCUSSION THREAD

**Discussion by MRchildNEO** [qihuju/qihuju.github.io#9] [Ideas]
1.  **卡牌游戏化**  角色扮演方式，以哲学的基本派别，思想实验和大观念为骨架，结合具体的内容情景进行主题讨论
2. **元胞自动机** 从一个母话题发起，在谈论中发现和形成有趣的话题，作为下一次房间的主题
3. **头脑风暴** 对讨论内容没有限定，规定发言者的提交的内容类型（豆瓣链接，观点，项目和人物）

**Comment by askender** [comment]
卡牌游戏化    不做评价。 我觉得「哲学的基本派别，思想实验和大观念」科普已经比较常见，「角色扮演」不知道怎么实现。设计成更具体的游戏机制会比较好。
元胞自动机 感觉和奇忽局在做的比较接近
头脑风暴    规定发言者的提交的内容类型（豆瓣链接，观点，项目和人物），这个不错，但提交形式还是语音的么？我一直想尝试语音和文字、以及多人协作的wiki同时推进的头脑风暴， 关键是风暴引来的第一滴水，是否能引发后续的衍生作品。

**Reply by Lupingyue** [reply]
> 元胞自动机 感觉和奇忽局在做的比较接近

对，目前的“脑图”蔓延生发的确有点元胞自动机的感觉，但是远远没有元胞自动机，Wolfram的模型那么酷。

**Reply by Lupingyue** [reply]
> 头脑风暴 规定发言者的提交的内容类型（豆瓣链接，观点，项目和人物），这个不错，但提交形式还是语音的么？我一直想尝试语音和文字、以及多人协作的wiki同时推进的头脑风暴， 关键是风暴引来的第一滴水，是否能引发后续的衍生作品。

这个构想听起来很酷，但是不知道最终是什么样的形态。

**Comment by askender** [comment]
歪个楼，
卡牌游戏化的简单设计，  比如： TRPG、mmorpg、各种桌游等

如果  只有哲学派别、思想实验、大观念等的卡牌会感觉发挥难度大。是否也将其设计为有某种功能性的卡牌，也可以靠取得胜利作为评判，也能省略不必要的语言沟通的部分。 做的过程中，可以不断平衡游戏性，让其成为一个注重策略的桌游/电子游戏。  如果加上无限分叉的共创添加的剧情，可以做成多分支的单人RPG

参考三国杀，狼人杀、七大奇迹等，设计一个回合制、非透明、非零和、有合作对抗特性的、有一定思想性和启发性的桌游：
角色牌：爱因斯坦（技能：相对论效应；） 李白（技能：千金散尽；）...
资源牌：各种类型的宇宙(可以不断发行新宇宙)；更务实的资源
技能牌：纵横、捭阖、降维、
时代牌：达到一定条件可以进入下一个时代；特殊条件下可以穿越到过去，技能受时代限制

**Reply by MingxuanXie** [reply]
> 资源牌：各种类型的宇宙(可以不断发行新宇宙)；更务实的资源 

这句话让我想到了前段时间很火的中国独立游戏 *戴森球计划*
进而联想到了我想尝试做的[星际穿越脑图](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-544970) 也许资源就是每个（speaker）节点的知识？节点间的相互链接形成星系（知识体系）及进一步发展成许煜说的*宇宙技术*
听上去有点像知识星球的开源+区块链+三维可视化版本

**Reply by Lupingyue** [reply]
> 歪个楼，
> 卡牌游戏化的简单设计， 比如： TRPG、mmorpg、各种桌游等
> 
> 如果 只有哲学派别、思想实验、大观念等的卡牌会感觉发挥难度大。是否也将其设计为有某种功能性的卡牌，也可以靠取得胜利作为评判，也能省略不必要的语言沟通的部分。 做的过程中，可以不断平衡游戏性，让其成为一个注重策略的桌游/电子游戏。 如果加上无限分叉的共创添加的剧情，可以做成多分支的单人RPG
> 
> 参考三国杀，狼人杀、七大奇迹等，设计一个回合制、非透明、非零和、有合作对抗特性的、有一定思想性和启发性的桌游：
> 角色牌：爱因斯坦（技能：相对论效应；） 李白（技能：千金散尽；）...
> 资源牌：各种类型的宇宙(可以不断发行新宇宙)；更务实的资源
> 技能牌：纵横、捭阖、降维、
> 时代牌：达到一定条件可以进入下一个时代；特殊条件下可以穿越到过去，技能受时代限制

柏舟这组建议很赞。特别是“加上无限分叉的共创添加的剧情，可以做成多分支的单人RPG”，跟目前奇忽局脑图生发的方式是可以呼应的。

**Reply by Lupingyue** [reply]
> > 资源牌：各种类型的宇宙(可以不断发行新宇宙)；更务实的资源
> 
> 这句话让我想到了前段时间很火的中国独立游戏 _戴森球计划_
> 进而联想到了我想尝试做的[星际穿越脑图](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-544970) 也许资源就是每个（speaker）节点的知识？节点间的相互链接形成星系（知识体系）及进一步发展成许煜说的_宇宙技术_
> 听上去有点像知识星球的开源+区块链+三维可视化版本

听起来很宏大，很期待你的第一版demo

**Reply by Lupingyue** [reply]
> > 资源牌：各种类型的宇宙(可以不断发行新宇宙)；更务实的资源
> 
> 这句话让我想到了前段时间很火的中国独立游戏 _戴森球计划_
> 进而联想到了我想尝试做的[星际穿越脑图](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-544970) 也许资源就是每个（speaker）节点的知识？节点间的相互链接形成星系（知识体系）及进一步发展成许煜说的_宇宙技术_
> 听上去有点像知识星球的开源+区块链+三维可视化版本

[许煜：定位、事件与宇宙技术](https://baijiahao.baidu.com/s?id=1595807833276909269&wfr=spider&for=pc)
「……若要使这宇宙技术思考得到兑现，我们必须将海德格尔的思想重新定位，并偏离 “夜的国度”的狭隘的空间，或者说将 “夜的国度” 重新定位。“定位”在我们这残酷的时代会被如何演绎呢？“夜的国度”不再被美国和苏联压迫着，**但却受到亚洲，或称之为“中华未来主义”的力量威胁。“中华未来主义”对无限制的技术加速发展持开放的态度。**这给我们在交叉路上再次思考的机会，而我们在思考过程中必须偏离宙斯的宇宙。这不只是技术哲学的问题，而且是哲学自身的问题。海德格于1936年命题《欧洲与德国哲学》的演讲中提出两个拯救欧洲的方案：1)防治欧洲人亚洲人的接触；2)征服欧洲人的无根性并防止共同体的瓦解。 我将他保守思想的言论交给读者自己衡量。」

@Tz-H 《重新定位“夜的国度”&“中华未来主义的威胁”》可以做“美国众神vs中国众仙”4
1，宇宙技术
2，夜的国度
3，中华未来主义
4，宙斯宇宙的偏离

**Reply by Lupingyue** [reply]
「也许资源就是每个（speaker）节点的知识？节点间的相互链接形成星系（知识体系）及进一步发展成许煜说的_宇宙技术」——每个speaker节点的知识「脑图」就是「资源牌」，这个很有趣， @Sleeper2060 说的[「脑图可以试试图像的机器学习，出来的影像可以做成GIF『NFT资源卡牌？』」](https://github.com/qihuju/qihuju.github.io/discussions/11#discussioncomment-553274)

**Comment by Lupingyue** [comment]
> * **元胞自动机** 从一个母话题发起，在谈论中发现和形成有趣的话题，作为下一次房间的主题

“元胞自动机”有很多玩法，从母话题中发现下一次房间的主题是一种玩法。这几天阿呆和李大维在讨论的“Wolfram-降临-圆相”的关联也可以产生一些有趣的玩法。

**Comment by Lupingyue** [comment]
> * **卡牌游戏化**  角色扮演方式，以哲学的基本派别，思想实验和大观念为骨架，结合具体的内容情景进行主题讨论

这个建议很同意。期待阿呆更多的细节。

**Comment by MRchildNEO** [comment]
> 1. **卡牌游戏化**  角色扮演方式，以哲学的基本派别，思想实验和大观念为骨架，结合具体的内容情景进行主题讨论
> 2. **元胞自动机** 从一个母话题发起，在谈论中发现和形成有趣的话题，作为下一次房间的主题
> 3. **头脑风暴** 对讨论内容没有限定，规定发言者的提交的内容类型（豆瓣链接，观点，项目和人物）

4. **语音切片**  语音及其时间轴作为呈现主体，把笔记，关键词，文献和发言人标记到对应语音片段，并展示出回应关系

**Reply by Lupingyue** [reply]
> 
> 1. **语音切片**  语音及其时间轴作为呈现主体，把笔记，关键词，文献和发言人标记到对应语音片段，并展示出回应关系

嗯，加了一个语音切片。阿呆的用词都很优雅。

**Comment by Tz-H** [comment]
关于（针对Clubhouse层面的）内容架构和规则约束制订方面的一些反向思考路径：

1. 一切规则都是反人性的，规则越详细人性越受到压抑和限制，靠激发创意来维持的项目和活动要特别小心这个陷阱。
2. 需要考虑言者的规训成本，以及因此丧失掉的潜在的新鲜血液（即言者新人）补充的可能性。
3. 需要考虑技术整合及支撑成本以及按照规则进行相关布属的人力/时间成本。
4. 听者的吸引力挑战，注意力维持，以及经由听者的二次传播的规划设计。
5. 言者在非物质回报、精神满足方面的规划设计。
6. 满足规则前提下的话题设计挑战。

最后，技术向人员的一个很普遍的问题或者心态就是一但手里有了一个锤子，就会开始满世界找钉子玩，进而看啥都像是钉子想锤几下，这种倾向需要特别小心，自省且收敛克制。
